test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -292,6 +305,24 @@ class test_ucp_sockaddr : public ucp_test {
         EXPECT_UCS_OK(status);
     }
 
+    static void rtag_complete_cbx(void *req, ucs_status_t status,
+                                  const ucp_tag_recv_info_t *tag_info,
+                                  void *user_data)
+    {
+        EXPECT_UCS_OK(status);
+    }
+
+    static void rtag_complete_check_data_cbx(void *req, ucs_status_t status,
+                                             const ucp_tag_recv_info_t *tag_info,
+                                             void *user_data)
+    {
+        mem_buffer UCS_V_UNUSED *recv_buffer = reinterpret_cast<mem_buffer*>(user_data);
+
+        if (status == UCS_OK) {
+            recv_buffer->pattern_check(1, recv_buffer->size());
+        }
```

Comment:
maybe print the status with UCS_TEST_MESSAGE?
need to know if we actually tried to validate